CodeGeneration/Sources/SyntaxSupport/TokenSpec.swift

```diff
@@ -222,10 +222,9 @@ public let SYNTAX_TOKENS: [TokenSpec] = [
   MiscSpec(name: "StringSegment", kind: "string_segment", nameForDiagnostics: "string segment", classification: "StringLiteral"),
   MiscSpec(name: "Unknown", kind: "unknown", nameForDiagnostics: "token"),
   MiscSpec(name: "Wildcard", kind: "_", nameForDiagnostics: "wildcard", text: "_"),
+  MiscSpec(name: "EndOfFile", kind: "eof", nameForDiagnostics: "end of file", text: "")
```

Comment:
Could you add the entry in alphabetical order? It just helps keep everything more organized.